cli/lib/index.js

```diff
@@ -31,7 +41,6 @@ prog
   .option('--secret-file-encoding <fileEncoding>', 'Encoding of secret file', prog.STRING)
   .option('-o, --output <outputFile>', 'Output to file', prog.STRING)
   .option('-O, --overwrite', 'Overwrites file if it already exists', prog.BOOL)
-  .option('--log-level <debug|verbose|info|warn|error|fatal>', 'log level', prog.STRING)
```

Comment:
Why did you removed this one?

Reply:
Since we can't delete `--verbose` from caporal, I've decided to drop the `--log-levels`, otherwise, it'll be very confusing to have 2 settings from log levels.